diff --git a/node_modules/@kingstinct/react-native-healthkit/kingstinct-react-native-healthkit.podspec b/node_modules/@kingstinct/react-native-healthkit/kingstinct-react-native-healthkit.podspec
index 1234567..89abcde 100644
--- a/node_modules/@kingstinct/react-native-healthkit/kingstinct-react-native-healthkit.podspec
+++ b/node_modules/@kingstinct/react-native-healthkit/kingstinct-react-native-healthkit.podspec
@@ -13,9 +13,13 @@
   s.platforms    = { :ios => "10.0" }
   s.source       = { :git => "https://github.com/kingstinct/react-native-healthkit.git", :tag => "#{s.version}" }
 
+  s.module_name  = "ReactNativeHealthkit"
   
   s.source_files = "ios/**/*.{h,m,mm,swift}"
+  s.exclude_files = "ios/ReactNativeHealthkit.swift", "ios/ReactNativeHealthkit.m", "ios/ReactNativeHealthkit-Bridging-Header.h"
   
 
   s.dependency "React"
+  load 'nitrogen/generated/ios/ReactNativeHealthkit+autolinking.rb'
+  add_nitrogen_files(s)
 end
