diff --git a/node_modules/@kingstinct/react-native-healthkit/ios/ReactNativeHealthkit.m b/node_modules/@kingstinct/react-native-healthkit/ios/ReactNativeHealthkit.m
index 6db47cc..4749f7c 100644
--- a/node_modules/@kingstinct/react-native-healthkit/ios/ReactNativeHealthkit.m
+++ b/node_modules/@kingstinct/react-native-healthkit/ios/ReactNativeHealthkit.m
@@ -110,6 +110,6 @@ @interface RCT_EXTERN_MODULE(ReactNativeHealthkit, RCTEventEmitter<RCTBridgeModu
 )
 
 RCT_EXTERN_METHOD(getWheelchairUse:(RCTPromiseResolveBlock)resolve
-                  reject:(RCTPromiseRejectBlock)reject)
+                  withRejecter:(RCTPromiseRejectBlock)reject)
 
 @end
diff --git a/node_modules/@kingstinct/react-native-healthkit/ios/ReactNativeHealthkit.swift b/node_modules/@kingstinct/react-native-healthkit/ios/ReactNativeHealthkit.swift
index 6d71ca2..fef0485 100644
--- a/node_modules/@kingstinct/react-native-healthkit/ios/ReactNativeHealthkit.swift
+++ b/node_modules/@kingstinct/react-native-healthkit/ios/ReactNativeHealthkit.swift
@@ -20,6 +20,8 @@ class ReactNativeHealthkit: RCTEventEmitter {
         self._runningQueries = Dictionary<String, HKQuery>();
         self._dateFormatter = ISO8601DateFormatter();
         
+        super.init()
+        
         if(HKHealthStore.isHealthDataAvailable()){
             self._store = HKHealthStore.init();
         }
